lib/matplotlib/backend_managers.py

```diff
@@ -17,17 +17,18 @@
 
 class ToolEvent(object):
     """Event for tool manipulation (add/remove)"""
-    def __init__(self, name, sender, tool, data=None):
+    def __init__(self, name, sender, tool, tool_name, data=None):
         self.name = name
         self.sender = sender
         self.tool = tool
+        self.tool_name = tool_name
         self.data = data
 
 
 class ToolTriggerEvent(ToolEvent):
     """Event to inform  that a tool has been triggered"""
-    def __init__(self, name, sender, tool, canvasevent=None, data=None):
-        ToolEvent.__init__(self, name, sender, tool, data)
+    def __init__(self, name, sender, tool, tool_name, canvasevent=None, data=None):
```

Comment:
This line fails PEP8 I think

Reply:
It does, you just have to look at the Travis build that failed because of this PEP8 error.  What's the point in having Travis if we don't use it.

Reply:
Don't worry! I used Travis.  I just wasn't 100% sure I'd remembered the line right, was too lazy to go back and check it, and was 99.9% sure @fariza could manage to figure it out.

Reply:
Ahh, okay.  I over reacted a bit though, I got an email to say that this PR had been approved, and I mistook that to mean it had been merged.  I haven't got used to this new review system yet...

Reply:
No problem!  I didn't approve it - I don't really understand it.